kratos/geometries/line_2d_2.h

```diff
@@ -883,6 +884,31 @@ class Line2D2 : public Geometry<TPointType>
         const double Tolerance = std::numeric_limits<double>::epsilon()
         ) override
     {
+        // The center of the geometry
+        const auto& r_center = this->Center();
+
+        // We compute the normal to check the normal distances between the point and the triangles, so we can discard that is on the triangle
+        const CoordinatesArrayType aux_point = ZeroVector(3);
+        const array_1d<double, 3> normal = this->UnitNormal(aux_point);
+
+        // We compute the distance, if it is not in the pane we
+        const Point point_to_project(rPoint);
+        Point point_projected;
+        const array_1d<double,3> vector_points = rPoint - r_center.Coordinates();
+
+        const double distance = GeometricalProjectionUtilities::FastProjectDirection(*this, point_to_project, point_projected, normal,  vector_points, 0);
```

Comment:
why you take normal to calculate the projection?

Reply:
The normal and the vector pointing to the center. It is the same we have done in the triangle